doc/auto-discovery-docker.md

```diff
@@ -1,49 +1,26 @@
 # Set Up Auto Discovery with Docker
 
-This is a guide on how to configure the ONVIF device service for automatic device discovery using Docke.
+This is a guide on how to configure the ONVIF device service for automatic device discovery using Docker.
 
 <img alt="overview" src="images/auto-discovery-docker-overview.jpg" width="75%"/>
 
-Note: For macOS, the network_mode: "host" probably does not work as expected: https://github.com/docker/for-mac/issues/1031
-
-## Non-Security Mode
-
-### Prepare edgex-compose/compose-builder
-
-#### 1. Download the [edgex-compose](https://github.com/edgexfoundry/edgex-compose) and setup it according to the [docker-compose setup guide](./docker-compose/README.md)
-
-#### 2. Update  the `add-device-onvif-camera.yml` file with the following content:
+## Prepare edgex-compose/compose-builder
+1. Download the [edgex-compose](https://github.com/edgexfoundry/edgex-compose)
+2. Change the directory to ` edgex-compose/compose-builder`
 
+## Update the `./compose-builder/add-device-onvif-camera.yml`
+Add subnets for discovering cameras:
 ```yaml
-version: '3.7'
-
 services:
   device-onvif-camera:
-    image: edgexfoundry/device-onvif-camera${ARCH}:${DEVICE_ONVIFCAM_VERSION}
-    ports:
-      - "127.0.0.1:59984:59984"
-    container_name: edgex-device-onvif-camera
-    hostname: edgex-device-onvif-camera
-    read_only: true
-    restart: always
-    networks:
-      - edgex-network
-    env_file:
-      - common.env
-      - device-common.env
+    ...
     environment:
-      SERVICE_HOST: edgex-device-onvif-camera
-      MESSAGEQUEUE_HOST: edgex-redis
-    depends_on:
-      - consul
-      - data
-      - metadata
-    security_opt:
-      - no-new-privileges:true
-    user: "${EDGEX_USER}:${EDGEX_GROUP}"
-
+      ...
+      APPCUSTOM_DISCOVERYSUBNETS: "192.168.1.0/24,10.0.0.0/24"
 ```
-> Example add-device-onvif-camera.yml contents
+See more settings at [configuration.toml](https://github.com/edgexfoundry/device-onvif-camera/blob/cba106416024542bb09af345da5a1638a965ef9b/cmd/res/configuration.toml#L107)
```

Comment:
Why `blob/cba106416024542bb09af345da5a1638a965ef9b` and not `main`?

Reply:
Modified. 
Change the description to specify the config section instead of using the permalink.
`See more settings at the [AppCustom section of the configuration.toml](https://github.com/edgexfoundry/device-onvif-camera/blob/main/cmd/res/configuration.toml)`